src/main/java/xyz/nucleoid/codecs/MoreCodecs.java

```diff
@@ -227,13 +233,13 @@ private static <T> List<T> unitArrayList(T t) {
      * @return a {@link MapCodec} that decodes the specified field
      */
     public static <T> MapCodec<T> propagatingOptionalFieldOf(Codec<T> codec, String name, Supplier<? extends T> defaultSupplier) {
-        return Codecs.createStrictOptionalFieldCodec(codec, name)
+        return codec.optionalFieldOf(name)
```

Comment:
We should probably entirely deprecate these now, I don't think there's any use of these as opposed to the normal optionalFieldOf

Reply:
I think it's fine to just deprecate them for now :slightly_smiling_face: